components/Navbar.vue

```diff
@@ -57,85 +71,98 @@ export default {
 </script>
 
 <style scoped>
-nav {
-  background: #1d1283;
+.navbar {
+  height: 73px;
   display: flex;
-}
-.hamburger-menu {
-  position: relative;
-  width: 35px;
-  height: 70%;
-  left: 10px;
-  flex-flow: column;
-  justify-content: space-around;
   align-items: center;
-  padding: 10px 0;
-  box-sizing: border-box;
-  cursor: pointer;
+  background-color: #1d1283;
+  color: white;
+  position: sticky;
+  transform-style: preserve-3d;
+  padding: 0 20px;
+  top: 0;
+  z-index: 1;
+}
+.hamburger {
   display: none;
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
 }
-.hamburger-menu span {
-  background-color: white;
-  width: 70%;
+.hamburger__bar {
   height: 3px;
-  margin: 2px 0;
+  width: 25px;
+  margin: 5px 0;
+  background-color: white;
```

Comment:
Can we please have constants for the colors used across the CSS and keep it consistent with Hex?